packages/packagers/js/src/index.js

```diff
@@ -2,9 +2,45 @@
 'use strict';
 
 import {Packager} from '@parcel/plugin';
+import fs from 'fs';
+
+const PRELUDE = fs.readFileSync(__dirname + '/prelude.js');
 
 export default new Packager({
   async package(bundle) {
-    return bundle.assets.map(asset => asset.output.code).join('\n\n');
+    let assets = bundle.assets
+      .map((asset, i) => {
+        let deps = {};
+
+        for (let dep of asset.dependencies) {
+          let resolvedAsset = bundle.assets.find(
```

Comment:
This isn't a great way to find the resolved ids for deps. It will be slower than necessary for sure. Perhaps another reason to have a bundle contain a Graph instead of a flat list of assets.

Reply:
For sure, I can start looking into that. We'll have to be able serialize/deserialize graphs since the packagers will be running in workers. Do you think that could possibly end up making it not worth it performance wise?